README.md

```diff
@@ -13,6 +11,20 @@ Additional information is available at the website: https://fwupd.org/
 
 The most up to date compilation instructions are available in the [Wiki](https://github.com/fwupd/fwupd/wiki/Compilation).
 
+**NOTE:** In most cases end users should never compile fwupd from scratch; it's a
+complicated project with dozens of dependencies (and as many configuration options)
+and there's just too many things that can go wrong.
+
+Users should just have fwupd installed and updated by their distro, hopefully
+by a packager who does understand all the config options.
```

Comment:
no need to throw shade at packagers

Reply:
hah, that was meant more with British sarcasm :) What about `Users should just have fwupd installed and updated by their distro, managed and tested by a trusted package maintainer."

Reply:
I don't think bringing a trusted vs untrusted packager into the equation matters.  Most people see distributions as an amorphous blob that provides a Linux OS.

Reply:
Maybe `Users should just have fwupd installed and updated by their distro, managed and tested by the package maintainer.`?

Reply:
That's better

Reply:
Okay, fixed.